bundles/org.openhab.binding.solarwatt/README.md

```diff
@@ -0,0 +1,154 @@
+# Solarwatt Binding
+
+Binding to query a [solarwatt](https://www.solarwatt.de/) [energy manager](https://www.solarwatt.de/energie-management/energymanager) and read the values of all attached devices.
+
+All supported values and devices were discovered while playing with my own energy manager.
+
+## Supported Things
+
+* EnergyManager  
+  Location  
+  Forecast  
+  GridFlow  
+  ProfileApp  
+  ScheduleApp  
+  SimpleSwitcher (seems deprecated)  
+  SmartEnergyManagement (seems deprecated)
+
+* BatteryConverter
+  
+* MyReserve  
+  MyReserveInverter  
+  MyReservePowerMeter
+  
+* EVStation (eg. Keba)
+* S0Counter
+* SunSpecInverter (eg. Fronius)  
+
+
+## Discovery
+
+You have to enter the hostname or ip-address of the energymanager itself.
+The attached devices and supported channels are discovered automatically.
+
+## Binding Configuration
+
+The binding has no configuration settings.
+
+## Thing Configuration
+
+Only the EnergyManager thing requires configuration.
+You have to give it the hostname or ip-address.
+
+## Channels
+
+### All devices
+
+| Channel Type ID | Item Type | Description |
+|-----------------|-----------|-------------|
+| stateDevice | String | value reflecting the state of the communication from the energy manager to the device. *ON* or *OFFLINE* |
+
+### EnergyManager
+
+| Channel Type ID | Item Type | Description |
+|-----------------|-----------|-------------|
+|timestamp | Number | Milliseconds since the epoch set to the last NTP time sync |
+|idTimezone | String | Timezone the energy manager is running in. All  timestamps are milliseconds since the epoch within this timezone |
+|fractionCPULoadTotal | Number:Percent | Total CPU load |
+|fractionCPULoadUser | Number:Percent | Userspace CPU load |
+|fractionCPULoadKernel | Number:Percent | Kernelspace CPU load |
+|fractionCPULoadAverageLastMinute | Number:Percent | Average 1 minute CPU load |
+|fractionCPULoadAverageLastFiveMinutes | Number:Percent | Average 5 minute CPU load |
+|fractionCPULoadAverageLastFifteenMinutes | Number:Percent | Average 15 minute CPU load |
```

Comment:
What's the purpose of the "fraction" prefix?

Reply:
The fraction prefix is used by the energymanager and I wanted to keep the names in sync.